src/test/EvenNumbersGeneratorTests.java

```diff
@@ -0,0 +1,54 @@
+package test;
+
+import com.company.EvenNumbersGenerator;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import static org.assertj.core.api.Assertions.*;
+
+public class EvenNumbersGeneratorTests {
```

Comment:
personally, it is hard to read these test names. I'd propose use `given_when_then` scheme or `method_scenario_expectation`

in this specific case I'd name :
```
 range1to3
```

```
range0to0
```

```
range1to1
```

....

Reply:
Renamed according method_scenario_expectation